scripts/run.sh

```diff
@@ -3,7 +3,6 @@
 trap cleanup INT
 
 function cleanup {
-	echo "Cleaning up services"
```

Comment:
I'd leave this in just so that users know that their ctrl-c has been received and that if the process hasn't exited it is because kill is taking a bit longer an expected.